libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFNetworkPlugin/SFNetworkPlugin.m

```diff
@@ -57,18 +58,27 @@ - (void) pgSendRequest:(CDVInvokedUrlCommand *) command
     NSString* path = [argsDict nonNullObjectForKey:kPathArg];
 
     NSDictionary* queryParams = [[NSDictionary alloc] init];
+    NSDictionary<NSString*, NSString*>* headerParams = [argsDict nonNullObjectForKey:kHeaderParams];
```

Comment:
Or to think of it another way, IMO that value drives a creation process of the request that includes setting the proper header values, not the other way around.